docs/features/pytorch.md

```diff
@@ -0,0 +1,140 @@
+- Install PyTorch using `pytorch` Conda channel (Legacy)
+
+With these options you can choose the best way to install PyTorch based on your requirements.
+
+## Installing from Conda-forge
+You can install PyTorch using the `conda-forge` channel.
+These are the community maintained builds of PyTorch.
+You can make direct use of the Nvidia provided packages to make sure the packages can work together.
+
+!!! note
+    Currently not well-supported for Windows, but there is lots of work being done to get this working.
+    Follow the work on the [feedstock](https://github.com/conda-forge/pytorch-cpu-feedstock)
+!!! note
+    Pixi uses the `system-requirements.cuda` to tell it can use the `cuda` packages.
+    Without it, pixi will install the `cpu` versions.
+
+=== "`pixi.toml`"
+    ```toml title="Bare minimum conda-forge pytorch with cuda installation"
+    --8<-- "docs/source_files/pixi_tomls/pytorch-conda-forge.toml:minimal"
+    ```
+=== "`pyproject.toml`"
+    ```toml title="Bare minimum conda-forge pytorch with cuda installation"
+    --8<-- "docs/source_files/pyproject_tomls/pytorch-conda-forge.toml:minimal"
+    ```
+
+To deliberately install a specific version of the `cuda` packages you can depend on the `cuda-version` package which will then be interpreted by the other packages at resolve time.
+Without specifying the `cuda-version` package, the latest version of cuda will be installed.
+=== "`pixi.toml`"
+    ```toml title="Add cuda version to the conda-forge pytorch installation"
+    --8<-- "docs/source_files/pixi_tomls/pytorch-conda-forge.toml:cuda-version"
+    ```
+=== "`pyproject.toml`"
+    ```toml title="Add cuda version to the conda-forge pytorch installation"
+    --8<-- "docs/source_files/pyproject_tomls/pytorch-conda-forge.toml:cuda-version"
+    ```
+
+With `conda-forge` you can also install the `cpu` version of PyTorch.
+A common use-case is having two environments, one for CUDA machines and one for none-CUDA machines.
+
+=== "`pixi.toml`"
+    ```toml title="Adding a cpu environment"
+    --8<-- "docs/source_files/pixi_tomls/pytorch-conda-forge.toml:use-envs"
+    ```
+=== "`pyproject.toml`"
+    ```toml title="Split into environments and add a CPU environment"
+    --8<-- "docs/source_files/pyproject_tomls/pytorch-conda-forge.toml:use-envs"
+    ```
+
+Now you should be able to extend that with your dependencies and tasks.
+
+Links to notable packages:
+
+- [pytorch](https://prefix.dev/channels/conda-forge/packages/pytorch)
+- [pytorch-cpu](https://prefix.dev/channels/conda-forge/packages/pytorch-cpu)
+- [pytorch-gpu](https://prefix.dev/channels/conda-forge/packages/pytorch-gpu)
+- [torchvision](https://prefix.dev/channels/conda-forge/packages/torchvision)
+- [torchaudio](https://prefix.dev/channels/conda-forge/packages/torchaudio)
+- [cuda-version](https://prefix.dev/channels/conda-forge/packages/cuda-version)
+
+## Installing from PyPi
```

Comment:
There have been reports where mixing certain conda packages that depended on PyTorch with PyPI pytorch seemed to be troublesome, maybe advise the user on that here.

Reply:
Added some more information, if you remember where those reports are, please send them in maybe I can elaborate some more based on them. But I can't find them.

Reply:
This is one: https://discord.com/channels/1082332781146800168/1312759176363249735

It gets hairy pretty quickly 😄 🚀

Reply:
Tried to improve the overall documentation based on some of the points in that chat.